packages/director/src/lib/hooks/reporters/slack.ts

```diff
@@ -1,46 +1,67 @@
-import { HookEvent, RunSummary, SlackHook } from '@sorry-cypress/common';
+import {
+CommitData,
+HookEvent,
+RunSummary,
+SlackHook
+} from '@sorry-cypress/common';
 import { getDashboardRunURL } from '@src/lib/urls';
 import axios from 'axios';
-import { shouldHookHandleEvent } from '../utils';
+import { isResultSuccessful,shouldHookHandleEvent } from '../utils';
 
 export async function reportToSlack({
   hook,
   runId,
   ciBuildId,
   runSummary,
   hookEvent,
+  commit: { branch, message },
 }: {
   hook: SlackHook;
   runId: string;
   ciBuildId: string;
   runSummary: RunSummary;
   hookEvent: HookEvent;
+  commit: CommitData;
 }) {
-  if (!shouldHookHandleEvent(hookEvent, hook)) {
+  if (!shouldHookHandleEvent(hookEvent, hook, runSummary, branch)) {
     return;
   }
-  let title = 'Test suite started';
-  let description = '';
-  if (runSummary.passes) {
-    description += ` ✅ passed ${runSummary.passes}`;
-  }
-  if (runSummary.failures) {
-    description += ` ❌ failed ${runSummary.failures}`;
-  }
-  if (runSummary.skipped) {
-    description += ` 👟 skipped ${runSummary.skipped}`;
-  }
 
-  if (hookEvent === HookEvent.RUN_START) {
-    title = '🚀 Started Run';
+  let title = '';
+  switch (hookEvent) {
+    case HookEvent.RUN_START:
+      title = `:rocket: *Run started* (${ciBuildId})`;
+      break;
+    case HookEvent.INSTANCE_START:
+      title = `*Instance started* (${ciBuildId})`;
+      break;
+    case HookEvent.INSTANCE_FINISH:
+      title = `*Instance finished* (${ciBuildId})`;
+      break;
+    case HookEvent.RUN_FINISH:
+      title = `${
+        isResultSuccessful(runSummary) ? ':white_check_mark:' : ':x:'
+      } *Run finished* (${ciBuildId})`;
+      break;
   }
 
-  if (hookEvent === HookEvent.INSTANCE_FINISH) {
-    title = 'Test suite finished';
-  }
+  const { passes, pending, skipped, failures } = runSummary;
+  const resultsDescription =
+    `${
+      passes > 0 ? ':large_green_circle:' : ':white_circle:'
+    } *Passed:* ${passes}\n\n\n` +
+    `${
+      pending > 0 ? ':large_yellow_circle:' : ':white_circle:'
+    } *Skipped:* ${pending}\n\n\n` +
+    `${failures + skipped > 0 ? ':red_circle:' : ':white_circle:'} *Failed*: ${
+      failures + skipped
+    }`;
+
+  const commitDescription = `*Branch:*\n${branch}\n\n*Commit:*\n${
+    message.length > 100 ? `${message.substring(0, 100)}...` : message
+  }`;
 
   if (hookEvent === HookEvent.RUN_FINISH) {
```

Comment:
@agoldis could you please explain why we need this 5 seconds timeout? Can't get it.

Reply:
```
  if (hookEvent === HookEvent.RUN_FINISH) {
    await new Promise((resolve) => {
      setTimeout(() => {
        resolve(null);
      }, 5000);
    });
  }
```

Reply:
I have removed it so far. Say if need to revert.

Reply:
Doesn't seem like those are required.